### DISCUSSION TITLE
Event filter drop-down select multiple

### DISCUSSION THREAD

**Discussion by rdmcguire** [blakeblackshear/frigate#1930] [Ideas]
It would be handy to have the ability to select multiple items in the filter inputs for events. I have a camera that generates pretty constant events, so when I visit events it is pages of "bird bird bird bird" as my wife likes to see what her chickens were up to during the day. All other categories are interesting to me, but it's pages of scrolling to get to it unless I select a different label, but then I have to look at labels one at a time. 

The ability to select all and then deselect one or more labels -- or simply select multiple labels individually -- would be a nice to have!

**Comment by jb2590** [comment]
This is a great idea, also the ability to select multiple clips to delete would be nice too. I find myself spending a lot of time selecting clips to delete where the camera moved slightly in the wind or something moves in frame and it re-detects my vehicle.

Thanks,
John
